src/components/dashboard/FaceVerification/screens/IntroScreen.js

```diff
@@ -75,6 +77,11 @@ const IntroScreen = ({ styles, screenProps }) => {
     navigate: navigateTo,
   })
 
+  const [, checkForBrowserSupport] = useBrowserSupport({
```

Comment:
only isIOS+non safari should be shown the pop in camera related screens

Reply:
@sirpy  i had a few telegram conversations when users tried to use unsupported browser. so i think we have to notify users about unsupported browsers on any platform

Reply:
@yaroslav-fedyshyn-nordwhale if we'll decide to check just for ios+safari on screens using camera, please create an separate hook which calls useBrowserSupport with custom onCheck:

```javascript
// src/components/browserSupport/hooks/useCameraSupport.js
import useBrowserSupport from "./useBrowserSupport"

import { isBrowser, isChrome, isIOSWeb, isMobileWeb, isSafari } from '../../../lib/utils/platform'

// if non-ios camera support is present on all browsers
// for IOS camera is supported omnly in Safari
export default (options = {}) => useBrowserSupport({ ...options, onCheck: () => !isIOSWeb || isSafari })
```